Dockerfile

```diff
@@ -18,16 +18,12 @@ RUN go get -d -v ./... \
     -ldflags '-s -w -extldflags "-static"' \
     -o /bin/solace_prometheus_exporter
 
-FROM registry.access.redhat.com/ubi9/ubi-minimal:latest
-
-
-RUN microdnf install -y curl-minimal-7.76.1-26.el9_3.2 libcurl-minimal-7.76.1-26.el9_3.2
-
+FROM golang:1.20.12-alpine3.19
```

Comment:
Does the Prometheus exporter need Golang SDK at RunTime? This image sounds very heavy to me...